.github/workflows/bench-turbopack.yml

```diff
@@ -2,16 +2,18 @@ name: Benchmark Turbopack
 
 on:
   push:
-    branches: [main, staging]
+    branches: [main]
```

Comment:
Do we need to run this on each push to `main` if we also have a scheduled run? (not sure if the scheduled benchmarking is the same)

Reply:
It's not the same. This benchmarks turbopack, the scheduled benchmarks all the other bundlers.
To get a continuous benchmarks for turbopack we run this on each main commit.

But I guess we could add a path filter for that.

Reply:
added